pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -503,25 +503,49 @@ func (t *FSTree) GetStream(addr oid.Address) (*objectSDK.Object, io.ReadCloser,
 
 // GetRange implements common.Storage.
 func (t *FSTree) GetRange(addr oid.Address, from uint64, length uint64) ([]byte, error) {
-	obj, err := t.Get(addr)
+	_, reader, err := t.getObjectStream(addr)
```

Comment:
effectiveness of ignoring an already decoded header is highly questionable to me

the header has payload len. So it makes sense to check `from` and `length` against it (as was done before)

btw, is `reader` limited by object payload len?